third_party/angle/tnt/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.19)
+project(angle)
+
+add_library(angle INTERFACE)
+target_include_directories(angle INTERFACE ${ANGLE_INCLUDE_DIR})
+
+# Uncomment the following lines and specify your folder containing ANGLE libraries to enable linking sample projects using ANGLE:
+# set(ANGLE_LIBS_DIR "")
+# target_link_directories(angle INTERFACE ${ANGLE_LIBS_DIR})
+# target_link_libraries(angle INTERFACE "libEGL.dll.lib" "libGLESv2.dll.lib")
```

Comment:
Was the goal here to support _running_ the samples or just _linking_/building them? This will not copy the DLLs into the binary of the samples.

Reply:
Didn't try but this might work on Windows (with Visual Studio)
```suggestion
# target_link_libraries(angle INTERFACE "libEGL.dll.lib" "libGLESv2.dll.lib")
# if WIN32
#     set_source_files_properties(libEGL.dll libGLESv2.dll PROPERTIES VS_DEPLOYMENT_CONTENT 1)
# endif
```

Reply:
[`VS_DEPLOYMENT_CONTENT`](https://cmake.org/cmake/help/latest/prop_sf/VS_DEPLOYMENT_CONTENT.html) property is for UWP only. Though we could come up with some solution by using [`SHARED IMPORTED`](https://cmake.org/cmake/help/latest/guide/importing-exporting/index.html#importing-libraries) targets.

Reply:
Commit link is broken